src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -411,4 +411,8 @@ public static void createFile(String... files) {
               }
             });
   }
+
+  public static final String[] snowflakeStringDataType = {
```

Comment:
This structure is for lookups, so I would suggest we use a Set not an array/list.